docs/Usage.md

```diff
@@ -158,3 +158,86 @@ class MyModule(torch.nn.Module):
 If registered module should be ignored by specific algorithms use `ignored_algorithms` parameter of decorator.
 
 In the example above, the NNCF-compressed models that contain instances of `MyModule` will have the corresponding modules extended with functionality that will allow NNCF to quantize, sparsify or prune the `weight` parameter of `MyModule` before it takes part in `MyModule`'s `forward` calculation.
+
+### Accuracy-aware model training
```

Comment:
I suggest adding the information about AA into the Key Features list in the main README with a reference to this doc.

Reply:
Done